G1ANT.Addon.Xlsx/G1ANT.Addon.Xlsx.Tests/Commands/xlsx.countrows/ShouldGetRowsCount_WhenSetMultipleRowsValues.G1ANT

```diff
@@ -0,0 +1,45 @@
+﻿addon xlsx version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
+
+♥folderPath = ♥environment⟦TEMP⟧\
+♥filePath = ♥folderPath + "TempTestDebug.xlsx"
+♥testValue = ‴ABC‴
+
+procedure DeleteTestFileIfExists
+    ⊂
+        if(System.IO.File.Exists(♥filePath))
+        {
+            System.IO.File.Delete(♥filePath);
+        }
+    ⊃
+end procedure
+
+procedure CreationWhenOpening
+    xlsx.open path ♥filePath createifnotexist true
+    xlsx.close
+end procedure
+
+procedure SetValuesInTestFile
+    xlsx.open path ♥filePath accessmode readwrite 
+    ♥Iteration = 4
+    while condition ⊂♥Iteration > 0⊃
+        xlsx.setvalue value ♥testValue row ♥Iteration colindex 1
+        ♥Iteration = ♥Iteration - 1
+    end while
+    xlsx.close
+end procedure
+
+procedure CountRowsFromTestFile
+    xlsx.open path ♥filePath accessmode readwrite   
+    xlsx.countrows result ♥rowCount
+    xlsx.close
+    test ⊂♥rowCount == 4⊃
+end procedure
+
+call DeleteTestFileIfExists
+call CreationWhenOpening
```

Comment:
We already discussed about that name. It does not contain a verb nor it describes what's happening inside.